NAMapKit.podspec

```diff
@@ -13,4 +13,5 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.dependency 'SDWebImage'
   s.dependency 'ARTiledImageView'
+  s.dependency 'pop'
```

Comment:
This dependency should not be added to NAMapKit, but to the demo project, right?